cinema_service/urls.py

```diff
@@ -4,5 +4,7 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls"), name="user"),
```

Comment:
The `include` function for the 'user' app should use the `namespace` argument instead of `name`. Change `name="user"` to `namespace="user"` to correctly namespace the included URLs.